### ISSUE TITLE
setCustomUserAttribute is not being updated on braze requests

### ISSUE THREAD

**Issue by smatias1989** [Appboy/appboy-android-sdk#133]
## What version of the Braze Android SDK are you using?
com.appboy:android-sdk-ui:3.5.0

## What did you do?

Action 1: setCustomUserAttribute
```
Appboy.getInstance(context).currentUser.setCustomUserAttribute("key_set_1", "value_1")
```
Wait 5 minutes: no braze request logged.

Action 2: addToCustomAttributeArray & logCustomEvent
```
Appboy.getInstance(context).currentUser.setCustomUserAttribute("key_set_2", "value_2")
Appboy.getInstance(context).currentUser.addToCustomAttributeArray("key_add", "value_add")
Appboy.getInstance(context).currentUser.logCustomEvent("event_name", eventProperties)
```
Wait 30 secs: braze request sent:
```
[{
  "sdk_version": "3.5.0",
  "events": [
    {
      "name": "add",
      "data": {
        "key": "key_add",
        "value": "value_add"
      }
    },
    {
      "name": "ce",
      "data": {
        "n": "event_name",
        "p": {
          // eventProperties
        }
      }
    }
  ]
}]
```

## What did you expect to happen?

For Action 1:
Braze request being fired with the custom attribute set by setCustomUserAttribute(key, value)

For Action 2:
Braze request fired do not miss attribute set by setCustomUserAttribute(key, value)

## Work around 1
Do: Action 1 & Action 2.
Wait for first braze request (missing the attributes).
Kill app. Open app again.
Braze request is sent:
```
[{
  "app_version": "1.0.4_1119",
  "device_id": ...,
  "time": 1565099156,
  "api_key": ...,
  "sdk_version": "3.5.0",
  "device": {
    "resolution": ...
  },
  "attributes": [
    {
      "custom": {
        "key_set_1": "value_1",
        "key_set_2": "value_2"
      }
    }
  ]
}]
```

## Work around 2

Downgrade to v3.4.0

For Action 2:
```
[{
  "sdk_version": "3.5.0",
  "attributes": [
    {
      "custom": {
        "key_set_2": "value_2"
      }
    }
  ],
  "events": [
    {
      "name": "add",
      "data": {
        "key": "key_add",
        "value": "value_add"
      }
    },
    {
      "name": "ce",
      "data": {
        "n": "event_name",
        "p": {
          // eventProperties
        }
      }
    }
  ]
}]
```
The custom attribute is send into `attributes` array in the request's first object body.

**Comment by radixdev** [comment]
Hi @smatias1989 , we're actively investigating and should have an update for you soon.

**Comment by radixdev** [comment]
Hello @smatias1989 ,

We've identified the issue and should have a fix available soon.

**Comment by smatias1989** [comment]
Awesome! Thanks for checking it so fast 😄

**Comment by radixdev** [comment]
Hello @smatias1989 ,

Thanks for filing this issue. This is now fixed in 3.6.0. Please see the [changelog](https://github.com/Appboy/appboy-android-sdk/blob/master/CHANGELOG.md) for information.

Cheers,
Braze Android Team
